README.md

```diff
@@ -8,6 +8,10 @@ Do you have other Github actions (Lighthouse, Cypress, etc) that depend on the N
 
 **Required** The name of the Netlify site to reach `https://{site_name}.netlify.app`
 
+### `extra_headers`
+
+Optional — Stringified HTTP Header object key/value pairs to send in requests
```

Comment:
It might be good to add a simple example in parentheses. `(eg. "{\"Authorization\": \"Bearer xyz\"}")` Not sure if I escaped all of the quotes on that, but you get the idea.